Gemfile.lock

```diff
@@ -1,23 +1,23 @@
 GEM
```

Comment:
There's quite a lot of dependencies being bumped here which seem unrelated to the commit they're in ('ignore GitHub links') – can we split them into a separate commit or PR?

Reply:
Yep, we can be more specific with this, I "just" copied the working Gemfile.lock in order to get to green,.